pallets/gear/src/tests.rs

```diff
@@ -954,7 +954,7 @@ fn block_gas_limit_works() {
         SystemPallet::<Test>::assert_has_event(
             Event::MessageDispatched(DispatchOutcome {
```

Comment:
I would suggest to merge @AndrePanin's PR with explanations! It also contains proper test changes